addon/utils/regexp.js

```diff
@@ -0,0 +1,7 @@
+const versionRegExp = /\d[.]\d[.]\d/;
+const shaRegExp = /[a-z\d]{8}/;
+
+export default {
```

Comment:
Any reason for exporting as a single object, rather than two named exports?  Since all of the places that the file is imported destructures both properties anyway, it seems marginally easier to instead do:

```js
import { shaRegExp, versionRegExp } from 'ember-cli-app-version/utils/regexp';
```

instead.